OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
What is the purpose of the adding the sphere mesh to this test? I believe the actual mesh is somewhat irrelevant to the test, but a sphere is particularly bad because it impossible to visually inspect the result for errors in orientation.  My preference would be to remove both sphere and cylinder for a mesh that is easy to visually distinguish a rotational error. Consider a box with differing half-lengths.